<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkStartOnTop">
     <property name="text">
      <string>Start with &quot;always on top&quot; on</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabConfigTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMainConfig">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbCharName"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCharacterName">
         <property name="text">
          <string>Character Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSlider" name="sldVolume">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QCheckBox" name="chkCriticalHits">
         <property name="text">
          <string>Show Critical Hits Seperate - Critical hits have a + added after the skill name. Ex: Stone Fist+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QCheckBox" name="chkCurrentDay">
         <property name="text">
          <string>Parse Current Day - This reads the current log as it comes in, aka Live.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblVolume">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="chkTrackPlayerOnStart">
         <property name="text">
          <string>Start with &quot;track player on map&quot; on</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="spnEventHours">
         <property name="suffix">
          <string/>
         </property>
         <property name="minimum">
          <number>6</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="lblEventhours">
         <property name="text">
          <string>Show events occuring in next x hours</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QCheckBox" name="chkKillingBlow">
         <property name="text">
          <string>Show Killing Blow skill - This adds an entry for every killing blow/death.</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QLabel" name="lblAppDataPath">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="btnChangeAppDataPath">
         <property name="text">
          <string>Change Appdata Dir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNotifications">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Notifications</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item row="2" column="0">
        <widget class="QGroupBox" name="grpWhisper">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Whisper Notification</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spnWhisperTime">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="chkWhisperSound">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Sound</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="lblLogonLogoffSpinboxTxt_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Popup Time(Secs)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="chkWhisperPopup">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Popup</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="grpSales">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Sales Notifications</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item row="0" column="2">
           <widget class="QSpinBox" name="spnSalesTime">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="chkSalesPopup">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Popup</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="chkSalesSound">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Sound</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="lblLogonLogoffSpinboxTxt_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Popup Time(Secs)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1" rowspan="3">
        <widget class="QGroupBox" name="grpLogonLogoff">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Logon/Logoff Notifications</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="2" column="0" colspan="4">
           <widget class="QGroupBox" name="grpLoginList">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Login Notification List</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="2" column="0" colspan="3">
              <widget class="QListWidget" name="lstLoginList"/>
             </item>
             <item row="0" column="0">
              <widget class="QLineEdit" name="txtLoginListInput"/>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="btnAddLoginList">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="btnRemoveLoginList">
               <property name="text">
                <string>Remove Selected</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="spnLogonLogoffTime">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="lblLogonLogoffSpinboxTxt">
            <property name="text">
             <string>Popup Time(Secs)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="chkLogonLogoffPopup">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Popup</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="chkLogonLogoffSound">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Sound</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="chkUseWhitelist">
            <property name="text">
             <string>Only on Listed</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPastLogs">
      <attribute name="title">
       <string>Past Logs</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupPast">
         <property name="title">
          <string>Past Logs</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QDateEdit" name="dateStart"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Start date for past logs</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>End date for past logs</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QDateEdit" name="dateEnd"/>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="chkPastDay">
            <property name="text">
             <string>Read Past Logs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Default Save Path for Exporting</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>125</height>
          </size>
         </property>
         <property name="title">
          <string>Performance Tuning - Probably shouldn't touch</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="spacing">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Max Display Lines</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtMaxDisplay"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Lines to clear when Full</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtLinesToClear"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblFileDateLabel">
         <property name="text">
          <string>File Date Format</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblLogDateTimeLable">
         <property name="text">
          <string>Log DateTimeFormat</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtLogDateTime"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtFileDate"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtDefaultSave"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
